pkg/handler-launcher-com/common.go

```diff
@@ -0,0 +1,38 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2019 Red Hat, Inc.
+ *
+ */
+package handler_launcher_com
+
+import (
+	"fmt"
+	"sort"
+)
+
+func GetHighestCompatibleVersion(serverVersions []uint32, clientVersions []uint32) (uint32, error) {
```

Comment:
i'm 99% sure this works, and I see you exercise this logic a bit in the server/client unit tests. I don't see a test that guarantees we pick the highest compatible version though. Can we get a test somewhere that covers that case too.